priprava/jazyky-nastroje/overeni.md

```diff
@@ -1,19 +1,11 @@
-## Ověření instalace
-
-Nejprve ověřme, že instalace nástrojů proběhla v pořádku. Ať už jste na jakémkoliv operačním systému, spusťte svoje nově nainstalované Visual Studio Code a z horní lišty vyberte _Terminal_ → _New Terminal_ (viz obrázek).
-
-::fig[VS Code Terminal]{src=assets/vscode-term.png}
```

Comment:
Je z nějakého důvodu smazán tento obrázek, nebo jsem to někde přehlédl?

Reply:
Díky, to je asi omyl